kpatch-build/kpatch-build

```diff
@@ -1174,9 +1210,14 @@ if [[ -n "$CONFIG_MODVERSIONS" ]]; then
 	trace_on
 fi
 
+if [[ -n "$CONFIG_LTO_CLANG" ]]; then
+	DIFF_OBJS="$TEMPDIR/thinlto_objs"
+else
+	DIFF_OBJS="$TEMPDIR/changed_objs"
+fi
```

Comment:
Why does lto need its own special file name?

Reply:
Initially, I thought we may still use changed_objs for something else. But I think we didn't use it at the moment. 

OTOH, "changed_objs" is not accurate for thin LTO, as not all these files are changed.